README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
+- Should be stable: no random failure
```

Comment:
True, but is this is not the test responsibility. Test should not try to cover random bugs though.

Reply:
Yes, but to be usable they must be reliable. If a test fails randomly it could be deactivated while the issue is fixed in the related service.

Reply:
Ok, this made things clearer, thanks.